registry/storage/driver/azure/azure.go

```diff
@@ -83,6 +99,26 @@ func New(accountName, accountKey, container, realm string) (*Driver, error) {
 		return nil, err
 	}
 
+	// Configure the http client for Storage server request since http.DefaultClient does not have optimal configuration
+	// On a busy server, we need to make many concurrent requests to the same storage server. The default MaxIdleConnsPerHost is 2,
+	// causing only 2 storage connections to be cached. As a result, the registry server needs to keep establishing new SSL connections to
+	// the storage server, causing bad perf. Also we explicitly specify other transport parameters here (for now they are pretty much
+	// the same as the default ones) in case the default parameter changes or doesn't fit our scenarios
```

Comment:
Also reference https://github.com/golang/go/issues/13801.

Reply:
Added reference to 13801